File: src/main/java/com/github/thorben/webnd/domain/user/UserRole.java
package com.github.thorben.webnd.domain.user;

public enum UserRole {

	USER {
		@Override
		public String pretty() {
			return "";
		}
	}, MODERATOR {
		@Override
		public String pretty() {
			return "(Moderator)";
		}
	};

	public abstract String pretty();

}

File: src/main/kotlin/com/github/thorben/webnd/domain/user/UserRole.kt
package com.github.thorben.webnd.domain.user

enum class UserRole {
	USER {
		override fun pretty(): String {
			return ""
		}
	},
	MODERATOR {
		override fun pretty(): String {
			return "(Moderator)"
		}
	};

	abstract fun pretty(): String
}